src/tools/imgtool/modules/thomson.cpp

```diff
@@ -693,7 +693,7 @@ static void thom_get_file(thom_floppy* f, unsigned head,
 		int i;
 		for ( i = 0; i < 8; i++ ) {
 	UINT8* data = thom_get_sector( f, head, track, firstsect + i );
-	stream_write( dst, data, f->sectuse_size );
+	dst->write(data, f->sectuse_size );
```

Comment:
You've half-adjusted the spacing around the parentheses in the call - either do it at both ends or don't do it at all.  Same goes for the rest of the changes in this file.